target/linux/mediatek/mt7622/base-files/etc/board.d/02_network

```diff
@@ -29,9 +30,28 @@ mediatek_setup_interfaces()
 	esac
 }
 
+mediatek_setup_macs()
+{
+	local board="$1"
+	local lan_mac=""
+	local wan_mac=""
+	local label_mac=""
+
+	case $board in
+	ruijie,rg-ew3200gx-pro)
+		lan_mac=$(macaddr_add $(mtd_get_mac_ascii u-boot-env ethaddr) 1)
```

Comment:
Now that WAN and label mac address are passed down by U-Boot (because the `ethernet0` alias is now present), I think it would be nicer to just read the address already assigned to `eth0` at this point instead of reading it from flash.

Reply:
Would using `$(macaddr_add $(get_mac_label) 1) ` make it more readable?